docs/cimpl6Authoring.md

```diff
@@ -98,69 +96,80 @@ Decide if you need to import any namespaces (e.g. OBF).
 
 Create your Class file with namespaces to import, and `Entry`, `Property`, `Abstract` and `Element` declarations.
 
-[CIMPL Language Reference Guide](cimpl6LanguageReference.md)
+[CIMPL Language Reference Manual](cimpl6LanguageReference.md#class-file)
 
 ## Map File
```

Comment:
A map file is only necessary if the user chooses not to use the CIMPL Base Classes (obf)

Reply:
Are you suggesting that this sentence be added?  I thought that a map file is necessary when using OBF if OBF doesn't cover the resources in the guide.

Reply:
_User-created_ maps files are necessary if OBF is _not_ used. If OBF _is_ used, then it is highly likely the user will _not_ have to create a map file.  (Add a note)

Reply:
OK added note.